code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -124,6 +124,20 @@
 	return ..()
 	//return QDEL_HINT_FINDREFERENCE
 
+/**
+ * start atmos processing based off the air on its turf. used mostly by vents & scrubbers
+ */
+/obj/machinery/atmospherics/proc/try_turf_atmos_process()
+	var/turf/open/turf = get_turf(src)
+	if (!istype(turf))
+		return
+
+	var/datum/gas_mixture/turf_gas = turf.air
+	if (isnull(turf_gas))
+		return
+
+	check_atmos_process(turf, turf_gas, turf_gas.temperature)
```

Comment:
This is a good idea, but let's make it more general!

Move this onto /atom, and put it in atmos_sensitive.dm
Also you should replace all the external calls to check_atmos_process() with this, except the one in LINDA_turf_tile.dm, that sucker is too hot for me to be comfortable wrapping it like this.

```suggestion
/**
 * Run when you update the conditions in which an /atom might want to start reacting to its turf's air
 */
 /atom/proc/atmos_conditions_changed()
	return
		
/atom/movable/atmos_conditions_changed()
	var/turf/open/open_loc = loc
	if (!isopenturf(open_loc)))
		return
	check_atmos_process(open_loc, open_loc.air, open_loc.air.temperature)

/turf/open/atmos_conditions_changed()
	check_atmos_process(src, air, air.temperature)
```

Reply:
> Also you should replace all the external calls to check_atmos_process()

Wait you mean throughout the whole code base or only in the air alarm interface? I assume you mean only when toggling the scrubber on via the air alarm else that would be way out of scope for this PR. 
Ok i'm not sure did you mean like everywhere?

Reply:
Ok so not many references and it's mostly localized to scrubber anyway
![Screenshot (204)](https://github.com/tgstation/tgstation/assets/110812394/065a3654-9c8c-4104-a429-a9df78100397)
Yup i can do this